crates/topos-node/src/lib.rs

```diff
@@ -0,0 +1,221 @@
+//! Temporary lib exposition for backward topos CLI compatibility
+use std::{path::PathBuf, process::ExitStatus};
+
+use futures::stream::FuturesUnordered;
+use futures::StreamExt;
+use opentelemetry::{global, sdk::metrics::controllers::BasicController};
+use process::Errors;
+use tokio::{
+    signal::{self, unix::SignalKind},
+    sync::mpsc,
+    task::JoinHandle,
+};
+use tokio_util::sync::CancellationToken;
+use topos_config::{
+    edge::command::BINARY_NAME,
+    genesis::Genesis,
+    node::{NodeConfig, NodeRole},
+};
+use topos_telemetry::tracing::setup_tracing;
+use topos_wallet::SecretManager;
+use tracing::{error, info};
+use tracing_opentelemetry::OpenTelemetrySpanExt;
+
+mod process;
+
+#[allow(clippy::too_many_arguments)]
+pub async fn start(
+    verbose: u8,
+    no_color: bool,
+    otlp_agent: Option<String>,
+    otlp_service_name: Option<String>,
+    no_edge_process: bool,
+    node_path: PathBuf,
+    home: PathBuf,
+    config: NodeConfig,
+) -> Result<(), Box<dyn std::error::Error>> {
+    println!(
```

Comment:
Why not `log::info!` this? Doesn't this make it trickier for the tools guys to capture the startup logs? Also: can we print the version here too? (Or are we doing that already somewhere else perhaps?)

Reply:
the `tracing` setup was define after, but it is fixed in another PR, this PR is just moving code around.